src/NuGet.Core/NuGet.Build.Tasks.Pack.Library/NuGet.Build.Tasks.Pack.Library.csproj

```diff
@@ -5,11 +5,14 @@
     <TargetFrameworks>netstandard1.3;net45</TargetFrameworks>
     <NoWarn>$(NoWarn);CS1591</NoWarn>
     <AssemblyName>NuGet.Build.Tasks.Pack</AssemblyName>
-	<RootNamespace>$(AssemblyName)</RootNamespace>
-    <PackageId>NuGet.Build.Tasks.Pack.Library</PackageId>
+    <RootNamespace>$(AssemblyName)</RootNamespace>
+    <PackageId>NuGet.Build.Tasks.Pack</PackageId>
```

Comment:
@rohit21agrawal is the circular dependency problem fixed now that the SDK no longer depends on pack?